src/services/myAccount.js

```diff
@@ -19,5 +19,35 @@ const getMyLeadListings = async params => {
   return Api.post(`${SERVICE_URL}/myAccount/getMyLeadListings`, params)
 }
 
-export { getMyCarListings, getMyLeadListings }
+const getAllUsers = async params => {
+  const Api = await apiAuth()
+  if (!Api) {
+    return
+  }
+
+  return Api.get(`${process.env.Dev_URL}/users`, params)
+}
```

Comment:
Please use. `SERVICE_URL` instead and check the logic of `SERVICE_URL`.
It gets updated from `.env ` file.